pkg/util/timeutil/time_zone_util.go

```diff
@@ -81,9 +82,6 @@ func TimeZoneStringToLocation(
 
 	tzOffset, ok := timeZoneOffsetStringConversion(locStr)
 	if ok {
-		if std == TimeZoneStringToLocationPOSIXStandard {
```

Comment:
Why was this check removed?

Reply:
because anything inside `timeZoneOffsetStringConversion` should be POSIX by default (unless it's a whole integer) -- I'll update the doc